jlc_kicad_tools/jlc_lib/cpl_fix_rotations.py

```diff
@@ -31,6 +33,13 @@
 _LOGGER = Log()
 
 
+@dataclass
+class DatabaseEntry:
+    rotation: int
+    offset_x: Optional[float] = None
```

Comment:
I would prefer just setting 0 offset on parsing if they don't exist, for consistency with rotation

Reply:
Sure.